app/react/Library/helpers/requestState.js

```diff
@@ -50,17 +50,18 @@ export function processQuery(params, globalResources, key) {
   };
 }
 
-export default function requestState(request, globalResources, calculateTableColumns = false) {
+export default function requestState(
+  request,
+  globalResources,
+  options = { calculateTableColumns: false, geolocation: false }
```

Comment:
For just two values, I believe we usually just used named properties, instead of equating them to options:

```
export default function requestState(request, globalResources, { calculateTableColumns = false, geolocation = false })
```
Something like that?

Reply:
Changed!

Reply:
I like this approach since you could omit the 3rd value if you don't need to pass any of the options. By using named properties the function will throw an error if I don't pass least an empty object.